webpack/scenes/Subscriptions/SubscriptionActions.js

```diff
@@ -75,7 +80,49 @@ export const loadSubscriptions = (extendedParams = {}) => async (dispatch) => {
   }
 };
 
+export const handleTask = task => (dispatch, getState) => {
+  const { pollingATask } = getState().katello.subscriptions;
+  if (pollingATask) {
+    return dispatch({ type: 'ALREADY_POLLING_TASK' });
+  }
+
+  return dispatch(pollTaskUntilDone(
+    task.id,
+    {},
+    POLL_TASK_INTERVAL,
+    task.input.current_organization_id,
+  // eslint-disable-next-line promise/prefer-await-to-then
```

Comment:
I don't see a reason to override the eslint rule here?  You can use a `try..finally` block with async/await (`catch` is not required.)